aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -118,36 +118,36 @@ module aptos_token::token {
     const ENO_BURN_TOKEN_WITH_ZERO_AMOUNT: u64 = 29;
 
     /// Withdraw proof expires
-    const EWITHDRAW_PROOF_EXPIRES: u64 = 29;
```

Comment:
let's make this 39 and leave the rest alone?

Reply:
updated